src/components/DataTable/index.js

```diff
@@ -1 +1,64 @@
-export * from './DataTable';
+import DataTable from './DataTable';
+import Table from './Table';
+import TableActionList from './TableActionList';
+import TableBatchAction from './TableBatchAction';
+import TableBatchActions from './TableBatchActions';
+import TableBody from './TableBody';
+import TableCell from './TableCell';
+import TableContainer from './TableContainer';
+import TableExpandHeader from './TableExpandHeader';
+import TableExpandRow from './TableExpandRow';
+import TableExpandedRow from './TableExpandedRow';
+import TableHead from './TableHead';
+import TableHeader from './TableHeader';
+import TableRow from './TableRow';
+import TableSelectAll from './TableSelectAll';
+import TableSelectRow from './TableSelectRow';
+import TableToolbar from './TableToolbar';
+import TableToolbarAction from './TableToolbarAction';
+import TableToolbarContent from './TableToolbarContent';
+import TableToolbarSearch from './TableToolbarSearch';
+
+DataTable.Table = Table;
```

Comment:
Do we really need to make all sub-components hard-dependencies of `<DataTable>`? Woud named exports below suffice?

Reply:
I consider this a temporary solution until we deprecate the old Table components from table v1. Once those are officially removed, then we expose all of these components as named exports like you said 😄

Reply:
Actually, thinking about this, should we just suffix them with `V2` so people can refer to the components themselves in the default exports?

Reply:
Yes I think it makes sense :+1: